### PR TITLE
Update a rewriting question

### PR DESCRIPTION
**Pull request by dangtuan1996pro**
image
https://dev.framgia.com/issues/184573?issue_count=64&issue_position=38&next_issue_id=184572&prev_issue_id=184574

### CODE REVIEW

**Comment by dangtuan1996pro** [comment]
ready

File: app/views/admin/writing/writing_questions/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            <% end %>
+          </div>
+          <div class="hr hr-16 hr-dotted"></div>
+          <!-- PAGE CONTENT ENDS -->
+        </div><!-- /.col -->
+      </div><!-- /.row -->
+    </div>
+
+    <div class="modal-footer no-margin-top">
+        <button class="btn btn-sm btn-danger pull-left" data-dismiss="modal">
```
Comment (nguyenducanhst96) [inline app/views/admin/writing/writing_questions/edit.html.erb]: ident

File: app/views/admin/writing/writing_questions/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <i class="ace-icon fa fa-times"></i>
+            <%= t ".close"%>
+        </button>
+        <button class="btn btn-sm btn-danger pull-left" type= "submit">
+            <i class="ace-icon fa fa-check"></i>
```
Comment (nguyenducanhst96) [inline app/views/admin/writing/writing_questions/edit.html.erb]: indent

File: app/views/admin/writing/writing_questions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            <% end %>
+          </div>
+          <div class="hr hr-16 hr-dotted"></div>
+          <!-- PAGE CONTENT ENDS -->
+        </div><!-- /.col -->
+      </div><!-- /.row -->
+    </div>
+
+    <div class="modal-footer no-margin-top">
+        <button class="btn btn-sm btn-danger pull-left" data-dismiss="modal">
```
Comment (nguyenducanhst96) [inline app/views/admin/writing/writing_questions/new.html.erb]: indent

File: app/views/admin/writing/writing_questions/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+            <i class="ace-icon fa fa-times"></i>
+            <%= t ".close"%>
+        </button>
+        <button class="btn btn-sm btn-danger pull-left" type= "submit">
+            <i class="ace-icon fa fa-check"></i>
```
Comment (nguyenducanhst96) [inline app/views/admin/writing/writing_questions/new.html.erb]: indent

**Pull request merged**